### PR TITLE
UI bug bash

### PR DESCRIPTION
**Pull request by jmar910**
### Description
Closes https://github.com/ConduitIO/conduit/issues/593
Closes https://github.com/ConduitIO/conduit/issues/590
Closes https://github.com/ConduitIO/conduit/issues/582
Closes https://github.com/ConduitIO/conduit/issues/578
Closes https://github.com/ConduitIO/conduit/issues/577

+ Added overflow scroll to connector overview
+ Added consistent disabled state styling to textarea
+ Fix pipeline list filtering "Running" and "Paused"
+ Add cursor pointer to pipeline detail dropdown "Remove"

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/conduit/pulls) for the same update/change.
- [x] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: ui/app/components/pipeline-editor.hbs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="flex bg-gray-300 absolute top-26 right-0 left-0 bottom-0">
   <div class="flex-auto" {{will-destroy this.unregister this.pipelineNodeManager}}>
     <div class="flex h-full relative">
-      <div class="bg-white w-76 flex-none shadow-md pl-9 pr-4">
+      <div class="bg-white w-76 flex-none shadow-md pl-9 pr-4 overflow-scroll">
```
Comment (jayjayjpg) [inline ui/app/components/pipeline-editor.hbs:4]: Nice quick fix! :sparkles:

File: ui/app/components/pipeline-editor.hbs [hunk 2]
```diff
@@ -36,14 +36,14 @@
 
                 <dd.Content class="bg-white rounded-md border border-gray-200 shadow-md text-sm">
                   <ul>
-                    {{!-- template-lint-disable no-invalid-interactive --}}
+                    {{! template-lint-disable no-invalid-interactive }}
```
Comment (jayjayjpg) [inline ui/app/components/pipeline-editor.hbs:39]: I wasn't aware of this formatting before, thank you for the update!

**Review by jayjayjpg — APPROVED** [review]
This reads great, thank you for the quick turnaround on all of these! :sparkles:

File: ui/app/components/pipeline-editor.hbs (hunk 2 above)
Comment (jmar910) [reply ui/app/components/pipeline-editor.hbs:39]: I wasnt either! my prettier on save just updated it lol

**Pull request merged**